webapp/components/_new/features/SearchResults/SearchResults.spec.js

```diff
@@ -0,0 +1,219 @@
+import { config, mount } from '@vue/test-utils'
+import Vuex from 'vuex'
+import SearchResults from './SearchResults'
+import helpers from '~/storybook/helpers'
+
+helpers.init()
+
+const localVue = global.localVue
+
+localVue.directive('scrollTo', jest.fn())
+
+config.stubs['client-only'] = '<span><slot /></span>'
+config.stubs['nuxt-link'] = '<span><slot /></span>'
+
+describe('SearchResults', () => {
+  let mocks, getters, propsData, wrapper
+  const Wrapper = () => {
+    const store = new Vuex.Store({
+      getters,
+    })
+    return mount(SearchResults, { mocks, localVue, propsData, store })
+  }
+
+  beforeEach(() => {
+    mocks = {
+      $t: jest.fn(),
```

Comment:
```suggestion
      $t: jest.fn((a) => a),
```

I suggest this one as a standard for testing, because then the translations are not empty and can be identified by their identifiers as the content.